ember-core/shared/src/main/scala/org/http4s/ember/core/Defaults.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s.ember.core
+
+import scala.concurrent.duration.DurationInt
+import scala.concurrent.duration.FiniteDuration
+
+private[ember] object Defaults {
+  val idleTimeout: FiniteDuration = 60.seconds
```

Comment:
We have different notations for such vals over the codebase, but maybe PascalCase better fits. wdyt?

Reply:
That's interesting. I saw camelCase and PascalCase vals in this project but how do you choose between eatch other ?